data/style.scss

```diff
@@ -42,6 +42,12 @@ body {
     word-break: break-word;
 }
 
+.footer {
+    text-align: center;
+    padding-top: 1.5rem;
+    color: var(--footer_color);
```

Comment:
How about `font-size: 0.7em;` here as well?

Reply:
Sounds good. Done in 474fadd.